cclib/parser/gaussianparser.py

```diff
@@ -323,10 +323,14 @@ def extract(self, inputfile, line):
             # Also, in older versions there is bo blank line (G98 regressions),
             # so we need to watch out for leaving the link.
             natom = 0
+            nfrozenatom = 0
             while line.split() and not "Variables" in line and not "Leave Link" in line:
                 natom += 1
+                if line.split()[1] == '-1':
```

Comment:
This is causing failures in other files (`water_ccsd.log`) where there aren't at least two tokens in a line.